animeal/res/ka.lproj/Localizable.strings

```diff
@@ -104,3 +106,5 @@
 "leaderBoard.empty"                               = "ლიდერბორდი აჩვენებს მოხალისეთა რეიტინგს";
 "verification.resendCode.titleTime"               = "კოდის ხელახლა გაგზავნა";
 "loginScreen.continueAsGuest"                     = "შესვლა სტუმრად";
+"loginScreen.registerOrLogin"                     = "";
```

Comment:
will business agree to put the texts at least from google translate I am sure AI these days will be good at this. if not please resolve these comments and merge.